src/Infrastructure/Persistence/Queries.hs

```diff
@@ -116,25 +115,32 @@ addContentWithTags content tags = transaction Serializable Write $ do
   Transaction.statement () $ add contentSchema [litContent content]
   Transaction.statement () $ add contentsTagsSchema (contentTag (litContent content) <$> (litTag <$> alreadyPresentTags) <> newTags)
 
--- SELECT USER BY USERNAME
+-- |
+-- Describes the possible error cases for queries that expect exactly one row as a result.
+data WrongNumberOfRows
```

Comment:
I would avoid using the word `Row` here, since it has too string relational database connotation (I know, we're talking about Postgres, here...) and use `Result` instead
```suggestion
data WrongNumberOfResults
```